build.gradle.kts

```diff
@@ -1,3 +1,10 @@
 plugins {
     id("com.automattic.android.publish-to-s3")
```

Comment:
I am afraid of this example being more harmful then helpful because we'll have `publishToS3` task in the root of the project which is not really what we want to use to publish the plugin itself. Maybe we can add that information as a comment?